state/lease/client.go

```diff
@@ -286,6 +286,17 @@ func claimLeaseOps(
 	return []txn.Op{claimLeaseOp}, newEntry, nil
 }
 
+// LookupLease returns a lease claim if it exists.
+// If it doesn't exist, expect to get an mgo.NotFoundError, otherwise expect to get
+func LookupLease(coll mongo.Collection, namespace, name string) (leaseDoc, error) {
+	var doc leaseDoc
+	err := coll.FindId(leaseDocId(namespace, name)).One(&doc)
+	if err != nil {
```

Comment:
if both the error and non-error condition return the same value, we really don't need an if statement right?